src/main/cpp/utilities/utils.h

```diff
@@ -55,7 +54,6 @@ char* sliceString( std::vector<char> inputBuffer, char* outputBuffer, int startO
 unsigned long sliceLong( std::vector<char> inputBuffer, int startOffset, bool littleEndian );
 
 // numbers to string
-#define SSTR( x ) dynamic_cast< std::ostringstream & >( \
- ( std::ostringstream() << std::dec << x ) ).str()
+#define SSTR( x ) std::to_string( x )
```

Comment:
C++11 provides a very convenient function to cast integers to strings overcoming the need for streams.